### PR TITLE
chore: Guidelines for operations after payment request.

### PR DESCRIPTION
**Pull request by alfmagsbidbax**
Guidelines for refund/capture etc.

### CODE REVIEW

File: docs/enrolment_and_payment.md [hunk 1]
```diff
@@ -132,3 +130,29 @@ Once a Cutoff request is received the `batchNumber` will increase and the new va
...
+* A partial capture has been performed and no other operations are to be performed on the payment.
+
+### Capture
+Must be performed 7 days within the Payment Request. The `paymentId` is used to identify the payment to be captured. Where 7 days is defined as `7*24 hours` from the time the Payment Request was sent.
+May be performed with a partial amount.
+
+> Note that in the case of a Payment Request being sent with `autoCapture` set to `true` the EPP will perform a Capture operation to finalize the payment immediately.
+
+### Refund
+May only be performed with an amount lower than previously captured amount.
```
Comment (adriantimberlid) [inline docs/enrolment_and_payment.md:155]: or equal to

File: docs/enrolment_and_payment.md [hunk 2]
```diff
@@ -132,3 +130,29 @@ Once a Cutoff request is received the `batchNumber` will increase and the new va
...
+## After the payment request
+After the payment request has resolved with the asynchronous confirmation callback, the Integrator can perform the following operations:
+Note that all operations require both a `paymentId` and a unique `messageId` to be performed.
+
+### Rollback
+By utilizing the rollback request the Integrator can cancel an ongoing payment. The `messageId` field is used to identify the payment to be rolled back. 
+
+General guidelines on when to rollback a payment: 
+
+* The Payment Request has been sent to the ePaymentPlatform but the Integrator has had technical issues receiving the callback *and* the Integrator believes it would be in the end customer's best interest to cancel the payment.
```
Comment (adriantimberlid) [inline docs/enrolment_and_payment.md:143]: to rollback the payment.

File: docs/enrolment_and_payment.md [hunk 3]
```diff
@@ -132,3 +130,29 @@ Once a Cutoff request is received the `batchNumber` will increase and the new va
...
+* The Payment Request has been sent but a `5xx`is received from the ePaymentPlatform.
+* The Payment Request has been sent and successfully been processed, but for whatever reason the goods or services are not delivered to the end customer as expected.
```
Comment (adriantimberlid) [inline docs/enrolment_and_payment.md:145]: I guess this would be cancel, not rollback.

File: docs/enrolment_and_payment.md [hunk 4]
```diff
@@ -132,3 +130,29 @@ Once a Cutoff request is received the `batchNumber` will increase and the new va
...
+* A partial capture has been performed and no other operations are to be performed on the payment.
```
Comment (adriantimberlid) [inline docs/enrolment_and_payment.md:146]: This might be a reversal, but not a rollback.

**Review by adriantimberlid — COMMENTED** [review]
Looks good! Added a couple of comments.

File: docs/enrolment_and_payment.md (hunk 3 above)
Comment (alfmagsbidbax) [reply docs/enrolment_and_payment.md:145]: Yes, I agree.

File: docs/enrolment_and_payment.md (hunk 1 above)
Comment (alfmagsbidbax) [reply docs/enrolment_and_payment.md:155]: Fixed!

File: docs/enrolment_and_payment.md (hunk 4 above)
Comment (alfmagsbidbax) [reply docs/enrolment_and_payment.md:146]: Moved to Cancel.

**Review by adriantimberlid — APPROVED** [review]

**Pull request merged**